samples/SkiaMultiplatformSample/src/awtMain/kotlin/org/jetbrains/skiko/sample/App.awt.kt

```diff
@@ -20,5 +22,68 @@ fun main() {
         skiaLayer.needRedraw()
         window.pack()
         window.isVisible = true
+        createWindowsJumpList()
+    }
+}
+
+/**
+ * The Jump List sample requires a separate executable for two reasons:
+ * 1. The java.exe process somehow blocks the Jump Lists so to enable them we would have to provide
+ *    an AppUserModelID for the sample's window, and
+ * 2. In the current implementation, the Jump List items launch the calling process when clicked with
+ *    command line arguments provided to the JumpListItem. Without a separate executable,
+ *    java.exe will be launched.
+ */
+private fun createWindowsJumpList() {
+    // Check if Jump Lists are supported
+    if (JumpList.isSupported()) {
+        // Start a Jump List building transaction
+        JumpList.build {
+            // If the process has an explicit AppUserModelID, it should be set for the Jump List
+            // before calling [beginList]
+            // setAppID("org.jetbrains.skiko.SkiaMultiplatformSample.awt")
+
+            // Create a new Jump List
+            beginList()
```

Comment:
I have this exception running via Gradle:
```
Exception in thread "AWT-EventQueue-0" java.lang.RuntimeException: Native exception in [Java_org_jetbrains_skiko_windows_JumpListBuilder_jumpList_1beginList], code 2147746301: Object is not connected to server 
Context: Failed to BeginList.
	at org.jetbrains.skiko.windows.JumpListBuilder.jumpList_beginList(Native Method)
	at org.jetbrains.skiko.windows.JumpListBuilder.beginList(JumpList.kt:133)
	at org.jetbrains.skiko.sample.App_awtKt$createWindowsJumpList$1.invoke(App.awt.kt:47)
	at org.jetbrains.skiko.sample.App_awtKt$createWindowsJumpList$1.invoke(App.awt.kt:41)
	at org.jetbrains.skiko.windows.JumpList.build(JumpList.kt:26)
	at org.jetbrains.skiko.sample.App_awtKt.createWindowsJumpList(App.awt.kt:41)
	at org.jetbrains.skiko.sample.App_awtKt.main$lambda$2(App.awt.kt:25)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```
Let's wrap the whole function into
```
        try {
            createWindowsJumpList()
        } catch (e: java.lang.RuntimeException) {
            println("Couldn't build the Jump List, most likely because the sample is run via java.exe. ${e.message}")
        }
```

Reply:
Thanks for finding this! Turns out it's not a good idea to uninitialise the COM apartment while the Jump List's COM object is alive in it 😅 Fixed!